### ISSUE TITLE
Initial loading of emojis takes some time

### ISSUE THREAD

**Issue by ovalseven8** [threema-ch/threema-web#734]
It's not a major problem, it just does not feel nice UX-wise. Perhaps one could load the emojis before you click on the tab for the first time?

**When I click on the emoji tab for the first time I have to wait a few seconds**
![bildschirmfoto_2019-01-26_09-21-48](https://user-images.githubusercontent.com/8258609/51784633-3e6be400-214c-11e9-8a36-47415d9a0211.png)

**Once it's loaded, everything is okay:**
![bildschirmfoto_2019-01-26_09-21-51](https://user-images.githubusercontent.com/8258609/51784644-5e030c80-214c-11e9-95cc-57253a7b32cd.png)

## Your Environment
* Threema Web version: 2.1.5
* Threema App version: 3.61
* Browser name and version: Firefox 64

**Comment by dbrgn** [comment]
Yes, that would probably be good. I'm not sure yet when and how to preload them though.
